core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.html

```diff
@@ -178,10 +178,10 @@ <h3>Subtopics</h3>
       <div ng-if="subtopicsListIsShown" class="oppia-mobile-collapsible-card-content">
         <div>
           <label>
-            <input class="practice-tab-checkbox protractor-test-toggle-practice-tab" type="checkbox" ng-model="editablePracticeIsDisplayed" ng-change="updatePracticeTabIsDisplayed(editablePracticeIsDisplayed)" ng-disabled="!topicRights.canEditTopic()">
+            <input class="practice-tab-checkbox protractor-test-toggle-practice-tab" type="checkbox" ng-model="editablePracticeIsDisplayed" ng-change="updatePracticeTabIsDisplayed(editablePracticeIsDisplayed)" ng-disabled="!topicRights.canEditTopic() || (!editablePracticeIsDisplayed && !topicHasMinimumQuestionsToPractice())">
```

Comment:
Why is `editablePracticeIsDisplayed` in the condition at all? Doesn't this input model it in the first place? As in `editablePracticeIsDisplayed` can only be true if `topicRights.canEditTopic() && topicHasMinimumQuestionsToPractice()`, right?

Reply:
What I was wanting to do here is allow the practice tab to be disabled no matter how many practice questions are present. So I disable the checkbox only when the practice tab is not displayed and the topic doesn't have enough questions. I thought of the case where we:
1. create 10 practice questions
2. set practice tab as displayed
3. remove a practice question
4. want to hide the practice tab

Does that make sense?

Reply:
So, in your case after step 4, as soon as it is flipped, the checkbox becomes disabled and cannot be re-enabled, right?

If what we want is for `editablePracticeIsDisplayed` to be false whenever number of questions go below threshold, I'd say add an error message below this as well informing the creator that there are fewer questions now than before and to recommend the checkbox be flipped, since otherwise there is no way for the creator to know to do this.
Also, it might be that somebody else deleted the question and if as soon as they flip this checkbox, it suddenly becomes disabled, they might be confused as well.

Reply:
Yes, it becomes disabled and cannot be re-enabled. And yep, makes sense -- I'll add a note when the practice tab should be turned off. Thanks!